src/xrGame/Level.cpp

```diff
@@ -549,6 +552,7 @@ void CLevel::OnFrame()
     {
         if (g_mt_config.test(mtLevelSounds))
         {
+            R_ASSERT(m_level_sound_manager);
```

Comment:
We don't need to check for `m_level_sound_manager`, but we should fix it's initialization on Linux 😉